recipes/vlfeat/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.9.20" %}
+
+package:
+  name: vlfeat
+  version: {{ version }}
+
+source:
+  fn: vlfeat-{{ version }}-bin.tar.gz
+  url: http://www.vlfeat.org/download/vlfeat-{{ version }}-bin.tar.gz
+  
+  patches:
+    - no_snprintf.patch  # [win and py35]
+    - no_crt_copy.patch  # [win]
+
+build:
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}                # [win]
```

Comment:
I'm not certain it's a bug, but it is certainly inconvenient.  We talked about this in today's call.  Your jinja is the best approach I've seen so far.  It makes logical sense to me to expect that the python version would be bound.  Can you think of any reason why one would want a different python than the one built with?  Maybe some build tool that runs only on py 27 (scons) but the output is py 3 OK?